manual/googlechrome-extensions/googlechrome-extension-template/googlechrome-.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-</id>
+    <version>1.0.0</version>
+    <title></title>
+    <authors></authors>
+    <owners>gep13 chocolatey</owners>
+    <projectUrl></projectUrl>
+    <iconUrl></iconUrl>
+    <projectSourceUrl></projectSourceUrl>
+    <docsUrl></docsUrl>
+    <bugTrackerUrl></bugTrackerUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/googlechrome-extensions/googlechrome-</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl></licenseUrl>
+    <description>
+####PACKAGING NOTE: This installs no software. It installs a registry key for the extension that Chrome will see and then ask you for permission to enable the extension if you are in Chrome or on the next run. Chrome will install the latest version of the extension. The mentioned version is the version at time of packaging, you can ignore it. Chrome will handle updates to the extension.
+    </description>
+    <summary>
+    </summary>
+    <tags>foss github chrome extension</tags>
```

Comment:
this should probably have the `admin` tag since we uses the local machine registry